orangecontrib/text/import_documents.py

```diff
@@ -20,13 +27,15 @@
 from pdfminer.layout import LAParams, LTTextBox, LTTextLine
 from bs4 import BeautifulSoup
 
+import serverfiles
+
 from Orange.data import DiscreteVariable, Domain, StringVariable
-from Orange.data.io import detect_encoding
+from Orange.data.io import detect_encoding, UrlReader as CoreUrlReader
+from Orange.data.util import get_unique_names
 from Orange.util import Registry
 
 from orangecontrib.text.corpus import Corpus
 
-
 DefaultFormats = ("docx", "odt", "txt", "pdf", "xml")
```

Comment:
Should .yaml also be specified here?

Reply:
No, it should not. These formats are 'documents' formats. Each file of these format represents a document. Metadata is considered differently, since it appends data to each document. Its formats are hardcoded in `_read_meta_data()`.